frontend/src/lib/components/AutheliaSetting.svelte

```diff
@@ -52,6 +52,10 @@
 				>
 				<input type="text" placeholder="yourorg" bind:value={org} />
 			</label>
+			<label class="block pb-2">
+				<span class="text-primary font-semibold text-sm">Display Name</span>
```

Comment:
here and in every similar field, we should say "Custom Name" instead of Display name

Reply:
to make it clear that it's non required

Reply:
should I add a `(optional)` in the placeholder as well or is it self-explanatory?

Reply:
It's self-explanatory I think